README.md

```diff
@@ -89,8 +97,41 @@ helm upgrade --install postfix-relay \
         --set smtp.relayPassword=${PASSWORD} \ 
         helm/postfix
 
+```
+
+## Chart Options for Managing Secrets
+
+1. Use existing secret:
+
+Use Other options such as [kubernetes external secrets](https://github.com/external-secrets/kubernetes-external-secrets) (see below) or [sealedSecrets](https://github.com/bitnami-labs/sealed-secrets)
+create the smtp password kubernetes secret then just reference it.
+
+```bash
+helm upgrade --install postfix-relay \
+        --set smtp.relayHost=${SMTP} \
+        --set smtp.relayMyhostname=my.local \
+        --set smtp.relayUsername=${USERNAME} \
+        --set smtp.relayPassword=${PASSWORD} \ 
+        --set useExistingSecret=true \
+        --set existingSecretName=postfix \
+        helm/postfix
+```
+
+1. Use kubernetes external secrets
```

Comment:
Can you add a link to the `ExternalSecret` resource type?

Reply:
it's there a few linws above - added link to CRD.yml